src/screens/QuestionScreen.tsx

```diff
@@ -79,90 +80,108 @@ function getElement(questionOption: QuestionOption) {
   switch (questionOption.type) {
     case QuestionOptionType.OPEN:
       return (
-        <OpenTextArea
-          placeholder={questionOption.extra_data.placeholder}
-          value={questionOption.answer?.values?.[0] ?? ''}
-          onChangeText={(value: string) => {
-            questionOption.answer = {
-              id: questionOption.answer?.id ?? 1,
-              values: [value],
-            } as Answer;
-          }}
-        />
+        <>
+          <QuestionTitle text={'Open Vraag'}></QuestionTitle>
```

Comment:
Het is geen 'vraag', het is een antwoordmogelijkheid en ik denk dat dat benoemen niet eens noodzakelijk is als je bovenaan de antwoordmogelijkheden (dus onder het lijntje van de vraag) zet "Antwoordmogelijkheden"

```suggestion
          <QuestionTitle text={'Open tekstveld'}></QuestionTitle>
```